source/content/guides/decoupled/drupal-nextjs-frontend-starters/04-preview.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Drupal + Next.js Frontend Starter for Front-End Sites
+subtitle: Implement Decoupled Preview
+description: Learn how to implement decoupled preview..
+tags: [webops, workflow, decoupled]
+contributors: [backlineint, cobypear, hckia, whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/drupal-nextjs-frontend-starters/preview
+anchorid: preview
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides information on how to implement Decoupled Preview.
+
+## Before You Begin
+
+- Make sure you have [set the variables for Decoupled Preview](/guides/decoupled/drupal-nextjs-frontend-starters/environment-variables#set-environment-variables-to-enable-decoupled-preview)
+
+## Configure Your Drupal Instance
+
+Your preview site and client secret must be correctly configured on your Drupal instance to use Decoupled Preview.
+
+1. Log in to your Drupal instance.
+1. Select **Configuration**, select **SimpleOAuth**, and then select **Clients**.
+1. Create a new consumer or use the default. Copy the UUID of the consumer you plan to use.
+1. Click **Edit** next to the consumer and add a new secret. Copy and save this value for later use.
+
+1. Select **Structure** and then **Preview Sites** to configure the preview
+site.
+
+1. Set the URL to point to `http(s)://{YOUR_SITE_URL}/api/preview`, replacing
+`{YOUR_SITE_URL}` with the URL of your Front-End Site, or `localhost:3000` to test the preview locally.
+
+1. Set a secret for the Preview Site and note this value down.
+
+Now you have all of the credentials needed to make authenticated requests to the Drupal instance, including the ability to preview content.
+
+## Use `getPreview`
```

Comment:
I'm not convinced that this section belongs here at all. If it was included, there should be a section prior to this that explains that preview for articles and pages will work without configuration if you use our starter projects. The examples below would only be necessary if they were customizing or implementing preview for other content types.